contribs/application/src/test/java/org/matsim/application/prepare/counts/CreateCountsFromBAStDataTest.java

```diff
@@ -0,0 +1,183 @@
+package org.matsim.application.prepare.counts;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.network.Link;
+import org.matsim.counts.Count;
+import org.matsim.counts.Counts;
+import org.matsim.counts.CountsReaderMatsimV1;
+import org.matsim.counts.MatsimCountsReader;
+import org.matsim.examples.ExamplesUtils;
+import org.matsim.testcases.MatsimTestUtils;
+
+import java.util.Map;
+
+public class CreateCountsFromBAStDataTest {
+
+	@Rule
+	public MatsimTestUtils utils = new MatsimTestUtils();
+
+	String carOutput = "car-test-counts.xml.gz";
+	String freightOutput = "freight-test-counts.xml.gz";
+
+	String ignoredCounts = "ignored.csv";
+	String manualMatchedCounts = "manual.csv";
+	String wrongManualMatchedCounts = "wrong_manual.csv";
+
+	String network = ExamplesUtils.getTestScenarioURL("berlin") + "network.xml.gz";
```

Comment:
It is better to use IOUtils.extendUrl(ExamplesUtils.getTestScenarioURL("berlin"), "network.xml.gz")